src/rbac.rs

```diff
@@ -94,17 +94,18 @@ impl Authorization {
             trace!(matches = false, "empty groups");
             return false;
         }
-        for rule in self.groups.iter() {
-            // If ANY rule matches, it is a match...
-            let mut rule_match = true;
-            for group in rule.iter() {
-                // We need ALL groups to match...
-                let mut group_match = true;
-                for mg in group.iter() {
+        for group in self.groups.iter() {
+            // Group typically has 1-3 elements (from,to,when)
+            // If ALL group matches, it is a match...
+            let mut group_match = true;
+            for rule in group.iter() {
+                // We can have multiple rules in a group, for example "Match ns=A,SA=B or ns=C"
+                // So we need ANY rule to match...
+                let mut rule_match = false;
+                for mg in rule.iter() {
```

Comment:
**Suggestion:** In the proto each individual item in the matches of a  `message Rules` is of type `message Match`. Could `mg` be changed to something more descriptive or could a comment be added explaining what mg means? I do see why calling the variable `match` in this scope could be undesirable and potentially unclear as well.